org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/lsp/JavaProtocolExtensions.java

```diff
@@ -111,4 +113,7 @@
 
 	@JsonRequest
 	CompletableFuture<List<SymbolInformation>> searchSymbols(SearchSymbolParams params);
+
+	@JsonRequest
+	CompletableFuture<List<? extends Location>> superMethod(TextDocumentPositionParams position);
```

Comment:
how about findLinks(FindLinksRequest request)
with FindLinksRequest {type="superImplementation", position}

that allows you to later implement findLinks({type="overrides", position})

Reply:
that should minimize the amount of custom client modifications